### DISCUSSION TITLE
CVE-2023-4807 Impact on Non-Windows Platforms

### DISCUSSION THREAD

**Discussion by rsbeckerca** [openssl/openssl#22300] [Q&A] [answered]
This CVE, as written at https://nvd.nist.gov/vuln/detail/CVE-2023-4807, suggests that there is a problem in the assembly code for the POLY1305 algorithm on Windows builds of OpenSSL.  Can we conclude that other builds of OpenSSL (for other platforms) that don’t use assembly at all, are not vulnerable to this problem, or is this something inherent in some x86 compiler generated code (c99, gcc, etc.) relating to XMM registers?

Also related, are non-Windows builds that use x86 assembly source affected, or is this only impacting Windows platform builds?

**Comment by kroeckx** [comment, marked as answer]
This is windows 64 bit specific. We didn't follow the ABI.

**Comment by rsbeckerca** [comment]
Thanks
